dadguide/models/monster_stats.py

```diff
@@ -90,9 +111,9 @@ def stats(self, monster_model, lv=99, plus=0, inherit=False, stat_latents: Monst
                 num_voice_awakening=monster_model.awakening_count(63)
```

Comment:
I guess this was already hard-coding the awakening number but can you fix this to use the awoken skills enum